packages/dialtone-vue2/components/list_item/list_item.vue

```diff
@@ -131,6 +132,15 @@ export default {
       type: Boolean,
       default: false,
     },
+
+    /**
+     * Class to apply to the wrapper element,
+     * note: it only applies to "default" type
+     */
+    wrapperClass: {
+      type: String,
```

Comment:
We should support all the standard class types which are `[String, Object, Array]`

Reply:
You're right, forgot about it.